src/features/allDataState/index.js

```diff
@@ -0,0 +1,297 @@
+import { createSelector } from 'reselect'
+
+const selectGroceriesStatus = (state) => state.groceries.dataStatus
+const selectProductsStatus = (state) => state.products.statusData
```

Comment:
bardzo podobne nazwy i cięzko rozroznic o co chodzi, jak ogarniesz selektory do statusow jak napisalem nizej to sie rozrozni lepiej

Reply:
konwencję nazewnictwa przyjmę od następnego projektu.
Pomijam.